src/System.Private.CoreLib/shared/Internal/Win32/RegistryKey.cs

```diff
@@ -416,7 +417,7 @@ public object GetValue(string name, object defaultValue)
                                 {
                                     Array.Resize(ref strings, stringsCount > 0 ? stringsCount * 2 : 4);
                                 }
-                                strings[stringsCount++] = toAdd;
+                                strings![stringsCount++] = toAdd;
```

Comment:
This should have a TODO-NULLABLE comment for https://github.com/dotnet/roslyn/issues/26761.  The `!` is only required right now because the compiler can't see that Array.Resize ensures `strings` is non-null.